packages/material-ui/src/Dialog/Dialog.js

```diff
@@ -190,7 +190,11 @@ class Dialog extends React.Component {
           {...TransitionProps}
         >
           <div
-            className={classNames(classes.container, classes[`scroll${capitalize(scroll)}`])}
+            className={classNames(
+              'mui-fixed',
```

Comment:
Looking at Bootstrap's implementation it seems `mui-fixed` should be here instead